### PR TITLE
Remove Saturn browser router

### PR DESCRIPTION
**Pull request by isaacabraham**
No longer needed.

### CODE REVIEW

File: Content/src/Server/ServerSaturn.fs [hunk 1]
```diff
@@ -19,10 +19,6 @@ let port = 8085us
 
 let getInitCounter () : Task<Counter> = task { return 42 }
 
-let browserRouter = scope {
```
Comment (mexx) [inline Content/src/Server/ServerSaturn.fs]: It's still used, when `Remoting` is requested

Comment (isaacabraham) [reply Content/src/Server/ServerSaturn.fs]: Doh. Good spot - will fix.

**Comment by theimowski** [comment]
Thanks! looks like the code could be further simplified, i.e. `mainRouter` now has a single `forward` -> could this be inlined?

**Comment by isaacabraham** [comment]
@theimowski yes, it could. I'm actually thinking that I was a little hasty with this - testing it out in Azure App Service it seems not to work!

**Comment by isaacabraham** [comment]
However I think that's an issue here on my branch. I'll confirm shortly.

**Comment by isaacabraham** [comment]
So there's a change needed in Saturn really that I've submitted a PR for. I can make a patch into the template to work around this temporarily.

**Pull request merged**